_maps/shuttles/pirate_geode.dmm

```diff
@@ -0,0 +1,1427 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"at" = (
+/obj/structure/grille,
+/obj/effect/spawner/structure/window/hollow/survival_pod,
+/obj/machinery/door/poddoor/shutters/preopen{
+	id = "geodebridge"
+	},
+/turf/open/misc/dirt{
+	planetary_atmos = 0;
+	initial_gas_mix = "o2=22;n2=82;TEMP=293.15"
```

Comment:
pls do not varedit gas mixes into maps

Reply:
Oops, must have missed a few tiles, thats supposed to be regular jungle dirt.